student.rb

```diff
@@ -1,15 +1,15 @@
-require "./person.rb"
+require './person'
 
 class Student < Person
-    def initialize(name = "Unknown", age, parent_permission == true, classroom)
-        super(name, age, parent_permission)
-        @classroom = classroom
-    end
+  def initialize(classroom)
+    super()
```

Comment:
- [ ] Same issue here, kindly add all the required parameters to the initialize method and also add the required parameters in the `super()`